code/LINDA/LINDA_turf_tile.dm

```diff
@@ -232,15 +233,14 @@
 	archived_cycle = SSair.times_fired
 
 /turf/simulated/proc/update_visuals(datum/gas_mixture/model)
-	overlays.Cut()
-	var/siding_icon_state = return_siding_icon_state()
-	if(siding_icon_state)
-		overlays += image('icons/turf/floors.dmi',siding_icon_state)
+	overlays -= gas_overlay

```

Comment:
you don't need to define a new var.

you can just do ```overlays -= SSair.plasma_overlay``` and ```overlays -= SSair.sleeptoxin_overlay```
and in in the ifs do ```overlays += SSair.plasma_overlay``` etc.